app/views/admin/articles/_individual_article.html.erb

```diff
@@ -1,55 +1,65 @@
-<style>
-  .article-body-html img {
-    max-width: 100%;
-  }
-</style>
-
 <% decorated_article = article.decorate %>
 
-<div class="card my-3"
+<article class="js-individual-article crayons-card p-6 flex flex-col gap-4"
      data-controller="article"
      data-article-id-value="<%= article.id %>"
      data-article-pin-path-value="<%= stories_feed_pinned_article_path %>"
      data-article-bg-highlighted-class="bg-highlighted"
      data-article-border-highlighted-class="border-highlighted"
      data-action="ajax:success@document->article#ajaxSuccess">
 
-  <div class="card-header">
-    <h2>
-      <a href="<%= article.path %>" target="_blank" rel="noopener">
+  <header>
+    <% if decorated_article.pinned? || article.featured || article.approved || (!article.published? && article.published_from_feed?) || article.video || article.user&.warned? %>
+      <div class="flex gap-1 mb-1">
+        <% if !article.published? && article.published_from_feed? %>
+          <span class="c-indicator c-indicator--danger">Unpublished</span>
+        <% end %>
+        <% if decorated_article.pinned? %>
+          <span class="c-indicator c-indicator--warning" data-testid="pinned-indicator">Pinned</span>
+        <% end %>
+        <% if article.featured %>
+          <span class="c-indicator c-indicator--success">Featured</span>
+        <% end %>
+        <% if article.approved %>
+          <span class="c-indicator c-indicator--info">Approved</span>
+        <% end %>
+        <% if article.video %>
+          <span class="c-indicator">Contains video</span>
+        <% end %>
+        <% cache "admin-user-info-#{article.user_id}-#{article.user&.updated_at}", expires_in: 4.hours do %>
+          <% if article.user&.warned? %>
+            <span class="c-indicator c-indicator--danger">User warned</span>
+          <% end %>
+        <% end %>
+      </div>
+    <% end %>
+
+    <% if article.main_image.present? %>
+      <a href="<%= article.path %>" target="_blank" rel="noopener" class="c-link c-link--branded mb-4" style="max-height:150px; max-width: 300px;">
+        <img src="<%= cloud_cover_url(article.main_image) %>" style="background: <%= article.main_image_background_hex_color %>;" alt="cover image" loading="lazy" class="block radius-default h-auto w-100" />
```

Comment:
```suggestion
      <a href="<%= article.path %>" target="_blank" rel="noopener" class="c-link c-link--branded mb-4">
        <img src="<%= cloud_cover_url(article.main_image) %>" style="background: <%= article.main_image_background_hex_color %>; max-height:150px; max-width: 300px;" alt="" loading="lazy" class="block radius-default h-auto w-100" />
```

(blocking) - I think the max-height/width attr need to be on the image rather than the link:
![image covering the article title](https://user-images.githubusercontent.com/20773163/153874357-cbf4381d-3dc3-43e2-8f11-b41a74054973.png)

(I've also sneaked in a little improvement to the image alt in the suggestion there 😄 )

Reply:
Fixed.